splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/MultiGroupGroupedAggregateOperationIT.java

```diff
@@ -488,7 +519,159 @@ public void testRollupAllOperations() throws Exception{
         }
         Assert.assertEquals("Not all groups found!", pairStats.size()+unameStats.size()+1,row);
     }
-	
+
+    @Test
+    public void testRollupWithNotNull_1() throws Exception {
```

Comment:
There are subtle differences, idea is to make compiler walk through various permutaions. I'll reflect that in one line comment in each test